start.php

```diff
@@ -176,7 +190,8 @@ function_exists( 'wp_is_json_request' ) &&
 	     $this_sdk_version != $fs_active_plugins->plugins[ $this_sdk_relative_path ]->version
 	) {
 		if ( $is_theme ) {
-			$plugin_path = basename( dirname( $this_sdk_relative_path ) );
+			// Saving relative path and not only directory name as it could be a subfolder
+			$plugin_path = $this_sdk_relative_path;
```

Comment:
This is creating an anomaly in logic. For plugins, we are saving the basename of the plugin, but for theme, we are saving the SDK's relative path! Can't we do better here? That is for both plugin and theme, save the basename?